scf/README.md

```diff
@@ -52,40 +56,61 @@ You are basically two `helm install's` away from deploying `SCF` with `Eirini`.
 
       # when using Bits-Service as image registry:
       BITS_SERVICE_SECRET: changeme
-      BITS_SERVICE_SIGNING_USER_PASSWORD: ehangeme
+      BITS_SERVICE_SIGNING_USER_PASSWORD: changeme
       BLOBSTORE_PASSWORD: changeme
     ```
    More information on the SCF deployment configurations can be found in the [SCF configurations docs](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#configuring-the-deployment)
 
-1. To deploy use the following commands as explained in the [SCF documentation](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#deploy-using-helm). In the commads `uaa` and `cf` are the helm charts you unzipped from the `eirini-scf-release`:
+1. To deploy `cd` into `eirini-release/scf/helm` and use the following commands as explained in the [SCF documentation](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#deploy-using-helm). In the commands `uaa` and `cf` refer to the helm chart directories from `eirini-release/scf/helm`:
+
     ```bash
-    $ helm install uaa --namespace uaa --values scf-config.yaml --name uaa
+    $ helm install uaa --namespace uaa --values scf-config-values.yaml --name uaa
     $ SECRET=$(kubectl get pods --namespace uaa -o jsonpath='{.items[?(.metadata.name=="uaa-0")].spec.containers[?(.name=="uaa")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}')
     $ CA_CERT="$(kubectl get secret $SECRET --namespace uaa -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
     $ helm install cf --namespace scf --name scf --values scf-config-values.yaml --set "secrets.UAA_CA_CERT=${CA_CERT}"
     ```
+
+   **NOTE**: When getting the uaa `SECRET`,
+be aware that secrets are versioned and the numerical suffix on the secret name will change if you upgrade the chart.
+When getting the value for `SECRET`, please check `helm list` or `kubectl get secrets --namespace uaa` for the correct name.
+
 1. Enjoy Eirini ;)
 
-### Cloud Specifics
+### Storage Class
 
-#### IBMCloud Container Service (Kubernetes)
+#### Using the HostPath Provisioner
 
-As storage class, you should deploy a `hostpath` provisioner to your cluster. You can for example follow the documentation in this [repository](https://github.com/MaZderMind/hostpath-provisioner#dynamic-provisioning-of-kubernetes-hostpath-volumes). The reason for this is that the `database` jobs in SCF are not working with the existing storage classes.
+As storage class, you can deploy a `hostpath` provisioner to your cluster. You can for example follow the documentation in this [repository](https://github.com/MaZderMind/hostpath-provisioner#dynamic-provisioning-of-kubernetes-hostpath-volumes). `hostpath` is not recommended for production use.
```

Comment:
Did you have success deploying Eirini without the hostpath provisioner? When we write this doc, we could not continue without, but this might have changed.

Reply:
yes I have a deployment that is using the default storage class from IKS.

However as mentioned in the README, the switchboard fails with som arp cache issues quite often which eventually goes away as the cache content expires. I believe the cleanup to the arp cache done in switchboard is the root cause as discussed in the slack conversation here:

https://cloudfoundry.slack.com/archives/C8RU3BZ26/p1546994880050000